src/streamFormat.cpp

```diff
@@ -630,7 +628,7 @@ StreamFormat LA_AVDECC_CALL_CONVENTION StreamFormatInfo::buildFormat_IEC_61883_6
 		default:
 			return getNullStreamFormat();
 	}
-	replaceField<16, 20>(fmt, static_cast<std::uint8_t>(0x0)); // fdf_evt = sampleFormat
```

Comment:
This is absolutly not an unused variable, but a mistake!!
fdf_evt variable is supposed to be used instead of the hardcoded 0x0 value!
(none of my compilers detected this as an unused variable though, not event gcc8)

Reply:
Interesting. On my system gcc 8.2.1 warns about this.

Reply:
Since I too, now have to use the lib on linux (was just compiled once a long time ago), I'll check which gcc version I have, and update the minimum requirements for gcc (either 8.2.1, or the one I have if it's lower, since it's working fine as well in my latest dev branch).
Thanks!